docs/modules/ROOT/pages/general-info/supported-platforms.adoc

```diff
@@ -6,20 +6,24 @@ details the JVMs currently supported by the Payara Platform Community (both Paya
 
 == Supported Java Virtual Machines
 
-The Payara Platform currently supports the following Java Virtual Machines:
+The Payara Platform currently runs the following Java Virtual Machines:
 
-* Oracle JDK8 (u162+), Oracle JDK 11 (11.0.5+)
-* Azul Zulu JDK8 (u162+), Azul Zulu JDK 11 (11.0.5u10+)
-* OpenJDK JDK8 (u162+), OpenJDK 11 (11.0.5+)
-* All other JVMs based on OpenJDK 8u162+ or 11.0.5+ (for example Corretto JDK, Liberica JDK, AdoptOpenJDK)
+* Azul Zulu JDK: 8 (u162+), 11 (11.0.5u10+)
+* Oracle JDK: 8 (u162+), 11 (11.0.5+)
+* Amazon Corretto: 8, 11 (11.0.5+)
+* Adopt Open JDK: 8, 11 (11.0.5+)
+* Adopt Open JDK with Eclipse Open J9
```

Comment:
We should specify JDK 8u162+ and JDK 11 for OpenJ9 as there exist multiple versions.

Reply:
I added the versions for Open J9 too, they are the same as for Hotspot Adopt Open JDK.